components/x-interaction/src/Hydrate.jsx

```diff
@@ -52,6 +52,12 @@ export function hydrate() {
 
 		const Component = getComponent(component)
 
+		if (!Component) {
```

Comment:
**praise:** This is fab and would have come in handy when I was dealing with some hydration issues with `x-live-blog-wrapper`